Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -561,31 +561,72 @@ - (void)testAddObjects {
     RLMUser *user = [self userForTest:_cmd];
     NSString *realmId = NSStringFromSelector(_cmd);
     RLMRealm *realm = [self openRealmForPartitionValue:realmId user:user];
+    NSDictionary *values = [AllTypesSyncObject values:1];
+
     if (self.isParent) {
         CHECK_COUNT(0, Person, realm);
+        CHECK_COUNT(0, AllTypesSyncObject, realm)
         RLMRunChildAndWait();
         [self waitForDownloadsForRealm:realm];
         CHECK_COUNT(4, Person, realm);
+        CHECK_COUNT(1, AllTypesSyncObject, realm);
+
+        AllTypesSyncObject *obj = [[AllTypesSyncObject allObjectsInRealm:realm] firstObject];
+        XCTAssertEqual(obj.boolCol,                           [values[@"boolCol"] boolValue]);
```

Comment:
weird formatting

Reply:
Instead of the alignment on values, should it just be natural line length?

Reply:
yes